common/riff/chunk.ksy

```diff
@@ -0,0 +1,17 @@
+meta:
+  id: chunk
```

Comment:
It's definitely not OK to name very format-specific things "chunk", as we're ending up getting `Chunk` class at top level, which will be shared across whole gallery. Please consider renaming this to `riff_chunk`?